remoulade/state/backends/postgres.py

```diff
@@ -118,7 +118,7 @@ def __init__(
     ):
         self.url = url or os.getenv("REMOULADE_POSTGRESQL_URL") or DEFAULT_POSTGRES_URI
         super().__init__(namespace=namespace, encoder=encoder)
-        self.client = client or sessionmaker(create_engine(self.url, pool_pre_ping=True))
+        self.client = client or sessionmaker(create_engine(self.url, pool_pre_ping=True, future=True))
```

Comment:
You should pass it as argument  in __init__ I think

Reply:
why I init ? In the doc they told about to add future flag in create_engine or sessionmaker

Reply:
in the init of PostgresBackend

Reply:
ok I will do it